do

```diff
@@ -27,10 +27,26 @@ check-version-bump() {
 help_package="Package the Helm chart"
 package() {
     mkdir -p target
-    cd target
+    cd target || return
```

Comment:
Is this `|| return` necessary? I feel like it's going to skip the packaging if we run into an error? But why would we hit an error if we `mkdir -p target` beforehand?

🤔

Reply:
This was a left over from some testing I was doing. I've removed it along with addressing a shellcheck warning.